commercetools/resource_cart_discount.go

```diff
@@ -622,16 +622,16 @@ func unmarshallCartDiscountStackingMode(d *schema.ResourceData) (platform.Stacki
 }
 
 func compareCartDiscountValidDates(old, new string) bool {
```

Comment:
Shall we make this an util function and rename it to `compareDateStrings()` or something?

Reply:
Sure 👍